metadata.json

```diff
@@ -12,7 +12,23 @@
     },
     {
       "name": "puppetlabs/docker",
-      "version_requirement": ">= 4.13.0 < 5.0.0"
+      "version_requirement": ">= 4.4.0 < 5.0.0"
+    },
+    {
+      "name": "puppetlabs/yumrepo_core",
```

Comment:
I think the convention is to not add the _core modules to metadata.json, because they're supposed to be vendored with the puppet agent. they are just not part of https://github.com/puppetlabs/puppet anymoe